subgraph.yaml.mustache

```diff
@@ -27,12 +27,10 @@ dataSources:
           file: ./abis/GPV2Settlement.json
         - name: ERC20
           file: ./abis/ERC20.json
-        {{#usesUniV2}}
         - name: UniswapV2Factory
           file: ./abis/UniswapV2Factory.json
         - name: UniswapV2Pair
           file: ./abis/UniswapV2Pair.json
-        {{/usesUniV2}}
```

Comment:
So, this conditional is not needed anymore?

Reply:
Yes - removed it on other places as well.